posthog/api/dashboard.py

```diff
@@ -138,6 +136,15 @@ def get_parents_query_dict(self) -> Dict[str, Any]:  # to be moved to a separate
         return {"team_id": self.request.user.team.id}
 
 
+class LegacyDashboardsViewSet(DashboardsViewSet):
+    legacy_team_compatibility = True
+
+    def get_parents_query_dict(self) -> Dict[str, Any]:
```

Comment:
What is the context for this change - did something break with the last PR?

Reply:
Ahh, didn't notice this method was supposed to be included with that last PR, shared dashboards don't load data now.

Reply:
#2514

Reply:
Wait, no, actually that is in master as it should've been, something else is at play 🤔

Reply:
I don't think I understand the question, there is no change, only an addition.